modules/dynamic-provisioning-azure-file-definition.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies
+//
+// * storage/dynamic-provisioning.adoc
+
+[id="azure-file-definition_{context}"]
+= Azure File object definition
+
+The Azure File StorageClass uses secrets to store the Azure storage account name
+and the storage account key required to create an Azure Files share. These
+permissions are created as part of the following procedure.
+
+.Procedure
+
+. Define a ClusterRole that allows access to create and view secrets.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+#  name: system:azure-cloud-provider
+  name: persistent-volume-binder-role
```

Comment:
Is this always the right name to use? If not, I'd make it a variable and add an annotation.